packages/KStepper/KStepper.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <ol class="k-stepper">
```

Comment:
Since `steps` could be an empty array, can we add a check here so that the `<ol>` element is not rendered if there are no steps?
```suggestion
  <ol v-if="steps && steps.length" class="k-stepper">
```